pom.xml

```diff
@@ -1229,11 +1262,11 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-compiler-plugin</artifactId>
-          <version>3.8.1</version>
+          <version>${maven.version}</version>
```

Comment:
The plugin version does not always comply with the Maven version. It is too dangerous and might lead to future build failures

Reply:
one could overwrite/force another version if it leads to failures on his site.. or do you mean inside citrusframework/citrus?

Reply:
The maven compiler plugin is not tied to the Maven version. The versions may differ so using the `${maven.version}` property here is not what we should do. Either stick with hardcoded version 3.8.1 or introduce another property like ${maven.compiler.version} and give it the value 3.8.1

Reply:
done 👍🏼